dingomata/cogs/text.py

```diff
@@ -281,14 +286,20 @@ async def waffle(self, ctx: discord.ApplicationContext) -> None:
 
     @slash(cooldown=True)
     @discord.option('user', description="Who to brush")
-    async def brush(self, ctx: discord.ApplicationContext, user: discord.User) -> None:
+    @discord.option('Body Part', description="What to brush", choices=["Head", "Ears", "Tail", "Paws", "Arms"])
+    async def brush(self, ctx: discord.ApplicationContext, user: discord.User, bodyPart: Optional[str],) -> None:
```

Comment:
nit: trailing comma. Also why are there less choices here than random choices, and they're capitalized differently?

Reply:
umm, there shouldn't be any differences, I copy and pasted the array from the brush options in the text response file.  The capitalize thing was because I was thinking since those options will be displayed to the user to pick from the first letter being capitalized might look nicer.  The plan was to then to just use .toLowerCase to fixed it.  I realize now I also forgot to put that function in, haha